app/Http/Controllers/Backend/QsController.php

```diff
@@ -34,21 +34,21 @@ public function getBatches(Request $request): \Illuminate\Http\Response
 
     public function markBatchesDone(Request $request): \Illuminate\Http\Response
     {
-        $batches = (array) $request->json()->get('batches');
+        $batches = (array) $request->json()->all('batches');
```

Comment:
This was wrong usage before and worked mostly by accident. Now, Symfony will throw an error on the wrong usage, so I was able to fix it.